### PR TITLE
Removed mentions of Gratipay, replaced with Bountysource.

### PR DESCRIPTION
**Pull request by bearbin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Other Stuff
 
 For other stuff, including plugins and discussion, check out the [forums](http://forum.mc-server.org) and [Plugin API](http://api-docs.cuberite.org).
 
-Support Us on Gratipay: [![gratipay shield](http://img.shields.io/gratipay/cuberite.svg)](https://www.gratipay.com/cuberite)
+[Support Us on Bountysource (salt)](https://bountysource.com/teams/cuberite)bountysource.com/teams/cuberite
```
Comment (SphinxC0re) [inline README.md]: When you look at the rich diff, this line looks stupid

Comment (bearbin) [inline README.md]: Partially because it's wrong. Now I have to download the repo and fix it.

Comment (SphinxC0re) [inline README.md]: Also, I think BS offers a badge. How about that?

Comment (bearbin) [inline README.md]: BS offers badges, but they're not relevant to the recurring-funding part that we're using, they're more for individual issues with bounties on them.

**Comment by SphinxC0re** [comment]
Also, you are not using the right URL. This one is the right one, if you are mentioning salt:
https://salt.bountysource.com/teams/cuberite

**Comment by bearbin** [comment]
Fixed.

**Pull request merged**